optuna/visualization.py

```diff
@@ -1,17 +1,24 @@
 from optuna.logging import get_logger
+from optuna.structs import StudyDirection
 from optuna.structs import TrialState
 from optuna.study import Study  # NOQA
 from optuna import type_checking
 
 logger = get_logger(__name__)
 
 if type_checking.TYPE_CHECKING:
+    from optuna.structs import FrozenTrial  # NOQA
```

Comment:
To follow [PEP-8 recommendation](https://www.python.org/dev/peps/pep-0008/#imports)(third party library imports should precede than own library imports, and those groups should be separated by a new line), please move this import as follows:
```python
    from plotly.graph_objs import Contour  # NOQA
    from plotly.graph_objs import Scatter  # NOQA
    from typing import List  # NOQA
    from typing import Optional  # NOQA
    from typing import Tuple  # NOQA

    from optuna.structs import FrozenTrial  # NOQA
```